client/src/components/ProgressWheel/ProgressWheel.jsx

```diff
@@ -0,0 +1,82 @@
+// https://codepen.io/bbrady/pen/ozrjKE
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import styles from './ProgressWheel.module.css';
+
+const ProgressWheel = ({
+  strokeWidth,
+  sqSize,
+  percentage,
+  numerator,
+  denominator,
+}) => {
+  // Size of the enclosing square
+  // const sqSize = this.props.sqSize;
+  // SVG centers the stroke width on the radius, subtract out so circle fits in square
+  const radius = (sqSize - strokeWidth) / 2;
+  // Enclose cicle in a circumscribing square
+  const viewBox = `0 0 ${sqSize} ${sqSize}`;
+  // Arc length at 100% coverage is the circle circumference
+  const dashArray = radius * Math.PI * 2;
+  // Scale 100% coverage overlay with the actual percent
+  const dashOffset = dashArray - (dashArray * percentage) / 100;
```

Comment:
In line 21 of SurveyCard.jsx, you multiply by 100 to get the percentage, then you divide by 100 here. You might as well store the percentage as a number between 0 and 1 (no 100's involved), and then you can write dashOffset = dashArray*(1-percentage).

Reply:
done